src/index.js

```diff
@@ -1 +1,66 @@
-console.log('Happy developing ✨')
+// 1
+function waitForAnyButtonToBeClicked(firstSelector, secondSelector) {
+  return new Promise(function (resolve, reject) {
+    const firstButton = document.querySelector(firstSelector);
+    const secondButton = document.querySelector(secondSelector);
+    if (!secondButton || !firstButton) {
+      reject();
```

Comment:
It often makes sense to do `return reject();` to avoid running the code below. Does not matter in this case, though.